src/components/StakingVaults/StakingVaultRow.tsx

```diff
@@ -115,7 +115,14 @@ export const StakingVaultRow = ({
           </SkeletonCircle>
         </Flex>
         <Skeleton isLoaded={isLoaded}>
-          <RawText size='lg' fontWeight='bold'>{`${asset.symbol} ${type}`}</RawText>
+          {/* TODO Remove USDC 0.3.0 test and special case when we have a migration path */}
```

Comment:
thanks for this @jdambron 

could you please implement this in a way such that we're not hard coding a check for the vault name here? e.g. if we have multiple vaults with the same symbol (yvUSDC) we do this conditional rendering, else do the normal rendering?

this just means when yearn releases 0.4.4 or 0.4.whatever we don't have special magic string conditions to update everywhere